src/lto/transactions/register.py

```diff
@@ -55,7 +55,11 @@ def to_binary(self):
 
     @staticmethod
     def __account_to_json(account):
-        return {'keyType': account['key_type'], 'publicKey': account['public_key']}
+        try:
+            if account['key_type']:
+                return {'keyType': account['key_type'], 'publicKey': account['public_key']}
```

Comment:
`self.accounts` should also contain data in a certain format. The cause of the bug isn't in this function, but in `__account_dict`.